### PR TITLE
fix: fix getPath

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
Fix the following error due to updating react-router-dom verison
```
react-dom.production.min.js:216 TypeError: Cannot read properties of null (reading 'loginRedirect')
    at data.ts:121:50
    at Array.every (<anonymous>)
    at k (data.ts:121:26)
    at SignIn.tsx:75:29
    at Fu (react-dom.production.min.js:262:359)
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at $o (react-dom.production.min.js:122:325)
    at Au (react-dom.production.min.js:261:308)
    at yu (react-dom.production.min.js:243:50)
    at react-dom.production.min.js:123:115
sc @ react-dom.production.min.js:216
data.ts:121 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'loginRedirect')
    at data.ts:121:50
    at Array.every (<anonymous>)
    at k (data.ts:121:26)
    at SignIn.tsx:75:29
    at Fu (react-dom.production.min.js:262:359)
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at $o (react-dom.production.min.js:122:325)
    at Au (react-dom.production.min.js:261:308)
    at yu (react-dom.production.min.js:243:50)
    at react-dom.production.min.js:123:115
```

## Test Plan
- Check if there's no above errors

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

### CODE REVIEW

File: webui/react/src/utils/experiment.test.ts [hunk 1]
```diff
@@ -6,9 +6,11 @@ import * as utils from './experiment';
 describe('Experiment Utilities', () => {
   describe('isExperiment', () => {
     it('should validate experiment tasks', () => {
-      const experimentTask = {
+      const experimentTask: Type.ExperimentItem = {
```
Comment (keita-determined) [inline webui/react/src/utils/experiment.test.ts:9]: this change is not related to the error, but I saw type errors in this file. fixed it!

File: webui/react/src/utils/experiment.test.ts [hunk 2]
```diff
@@ -30,7 +32,7 @@ describe('Experiment Utilities', () => {
         name: 'Count Active Processed',
         resourcePool: 'cpu-pool',
         startTime: '2021-11-29T00:00:00Z',
-        state: Type.CommandState.Assigned,
+        state: Type.CommandState.Queued,
```
Comment (keita-determined) [inline webui/react/src/utils/experiment.test.ts:35]: `Assigned` doesnt exist anymore

**Comment by hamidzr** [comment]
could you please add a unit test that showcases the issue?

**Review by hamidzr — APPROVED** [review]
lgtm.

**Comment by hamidzr** [comment]
also this is kinda odd the way we're getting the redirectValue https://github.com/hamidzr/determined/blob/552bef9c4737f0d882ea4f8915688a341284c005/webui/react/src/pages/SignIn.tsx#L75 at least with the current type info we have.

**Pull request merged**